src/Validate.php

```diff
@@ -24,6 +24,13 @@
  */
 class Validate
 {
+    /**
+     * Validation for the secret string
+     *
+     * @var string
+     */
+    public static $secretPattern = '/^.*(?=.{12,}+)(?=.*[0-9]+)(?=.*[A-Z]+)(?=.*[a-z]+)(?=.*[\*&!@%\^#\$]+).*$/';
```

Comment:
Leave by default a validation that is more flexible, if a person needs to restrict it it will be available, but I don't see a reason for it

Reply:
Got it.